lib/crypto/butil.js

```diff
@@ -40,12 +40,21 @@ module.exports.unbuffer = function unbuffer(object, inplace) {
   return copy
 }
 
-module.exports.bufferize = function bufferize(object, inplace) {
+module.exports.bufferize = function bufferize(object, inplace, exceptions) {
```

Comment:
Maybe we ought to change the signature to `bufferize(object, options)` now that we've got more than one option... not necessarily now though.